apps/api_web/test/api_web/api_controller_helpers_test.exs

```diff
@@ -112,6 +112,15 @@ defmodule ApiWeb.ApiControllerHelpersTest do
       params = %{"stop" => nil}
       assert ApiWeb.ApiControllerHelpers.filter_valid_field_params(params) == %{"stop" => []}
     end
+
+    test "can provide fields for multi-word types" do
+      params = %{"live_facility" => "updated_at", "route_pattern" => "name,typicality"}
```

Comment:
This should be `live-facility` to match the URL and `type` value for the resource.

Reply:
Currently, the JSON-API library we use expects it to be `live_facility`. I believe this is set by `config :ja_serializer, key_format: :underscored` in `apps/api_web/config.exs`.

Reply:
Or does it just use types that we set somewhere?

Reply:
We must be setting it somewhere if it's different for live facilities and route patterns.